Lib/dis.py

```diff
@@ -338,8 +338,11 @@ def _get_instructions_bytes(code, varnames=None, names=None, constants=None,
                 argval, argrepr = _get_const_info(arg, constants)
             elif op in hasname:
                 argval, argrepr = _get_name_info(arg, names)
+            elif op in hasjabs:
+                argval = arg*2
+                argrepr = "to " + repr(argval)
```

Comment:
Why not go a step further and divide the labels in dis output by 2?

Reply:
A couple of reasons.
* I wanted to keep changes to a minimum.
* There is no guarantee that instructions will remain the same size, so using byte offsets keeps things more consistent across versions.

Reply:
Sure. However now we have a confusing discrepancy between what's displayed and how we're encouraged to think about the bytecode in the C code. I guess there's no perfect answer, so status quo wins.